apps/generator-cli/src/README.md

```diff
@@ -231,14 +231,14 @@ more information about the possible arguments and a detailed usage manual of the
 
 ```sh
 npm install @openapitools/openapi-generator-cli@previous
-npm i @openapitools/openapi-generator-cli@1.0.18-4.3.1                                         
+npm i @openapitools/openapi-generator-cli@2.9.0                                         
 ```
 
 or using yarn
 
 ```sh
 yarn add @openapitools/openapi-generator-cli@previous
-yarn add @openapitools/openapi-generator-cli@1.0.18-4.3.1
+yarn add @openapitools/openapi-generator-cli@2.9.0
```

Comment:
these shouldn't be changed as these are instructions for installing previous version (1.x)

i'll file a new one to update the above.